changelog/6276.improvement.rst

```diff
@@ -0,0 +1 @@
+Allow Rasa to boot even when UnsupportedDialogueModelError or UnsupportedModelError occurs. Forward HTTP Error responses to standard log output.
```

Comment:
We recently changed our docs system 🙈  The changelog entries now have to be in `md` format.